docs/changelog/2024.md

```diff
@@ -16,6 +16,23 @@
 .. role:: small
 ```
 
+## 2024-11-13 {small}`hub 0.30.1`
```

Comment:
Let's just use 0.30 here. The patch number doesn't make much sense yet.

Reply:
Used it to test if laminci failure was due to an existing tag.